lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,447 @@
+        <li>The value of actor must be in the form of `valueReference`</li>
+        <li>Only the following system and code combinations may be used in the performer function `coding`</li>
+        <ul>
+          <li>http://terminology.hl7.org/CodeSystem/v2-0912 - OP</li>
+          <li>http://terminology.hl7.org/CodeSystem/v3 - PRF</li>
+          <li>http://terminology.hl7.org/CodeSystem/v3 - VRF</li>
+        </ul>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer
+
+  - name: reason
+    required: 'No'
+    type: CodeableConcept
+    description: Diagnosis codes for why the charge was rendered.
+    action: modify
+    example: |
+      {
+        "reason": [
+          {
+            "extension": [
+              {
+                "valueUnsignedInt": 1,
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+              }
+            ],
+            "coding": [
+              {
+                "extension": [
+                  {
+                    "valueCoding": {
+                      "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                      "code": "3693"
+                    },
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                  }
+                ],
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+
+  - name: replacing
+    required: 'Yes'
+    type: Extension
+    description: The resource that this ChargeItem is replacing.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueId": "1",
+            "url": "item-version"
+          }
+        ],
+        "valueReference": {
+          "reference": "ChargeItem/317370800"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+        <li>The item-version sub extension is required</li>
+      </ul>
+    url:  https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing?_format=json
+
+  - name: performing-location
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: modify
+    example: |
+      {
+        "valueReference": {
+          "reference": "Location/21304903",
+          "display": "NM"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
```

Comment:
For consistency, I think we should include a note detailing `Must be a Location reference`

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/e34c85b8dc4b0dce25552a7b3c89cbd55ce69b82#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R289-R292